firedrake/function.py

```diff
@@ -19,7 +21,7 @@
     cachetools = None
 
 
-__all__ = ['Function', 'interpolate']
+__all__ = ['Function', 'interpolate', 'PointOutOfDomainError', 'make_c_evaluate']
```

Comment:
I don't have particularly strong feelings for any variation. I will change if you prefer that way.